cypress/integration/examples/location.spec.js

```diff
@@ -12,9 +12,11 @@ context('Location', () => {
 
   it('cy.location() - get window.location', () => {
     // https://on.cypress.io/location
-    cy.location().should((location) => {
+    cy.location().should(location => {
       expect(location.hash).to.be.empty
-      expect(location.href).to.eq('https://example.cypress.io/commands/location')
+      expect(location.href).to.eq(
+        'https://example.cypress.io/commands/location'
+      )
       expect(location.host).to.eq('example.cypress.io')
       expect(location.hostname).to.eq('example.cypress.io')
       expect(location.origin).to.eq('https://example.cypress.io')
```

Comment:
move `example.cypress.io` to a local variable and use string interpolation for this line to make the code cleaner

Reply:
file deleted since this is sample test only